db/migrate/20180521163838_create_orders.rb

```diff
@@ -0,0 +1,24 @@
+class CreateOrders < ActiveRecord::Migration
+  def change
+    create_table :flowers do |t|
+      t.string :name
+      t.integer :price
+      t.timestamps
+    end
+ 
+    create_table :customers do |t|
+      t.string :name
+      t.string :address
+      t.string :email
+      t.timestamps
+    end
+ 
+    create_table :orders do |t|
+      t.belongs_to :customer, index: true
+      t.belongs_to :flower, index: true
+      t.string :status
+      t.integer :quantity
+      t.timestamps
+    end
```

Comment:
Why create 3 tables in a migration called `CreateOrders`?

Reply:
Because Orders belongs to the other 2 models. I think better way would have been to either create 3 migrations or name the migration differently?